tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
+    - items to evaluate technological and architectural health of projects
+- Community outreach on any items within scope, for instance, social media/conferences/podcasts/meetups et cetera
+- Collaboration with other sustainability efforts
+
+### Out of scope
+- Form an umbrella organization beyond the CNCF
+- Establish a compliance and standards body beyond the CNCF space
+- Evaluate individual company infrastructures
+- Focus outside of Cloud Native Technologies, according to the [CNCF Cloud Native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md)
+
+## Deliverables to ToC
+- landscape for the carbon and energy metrics/measurements/management savings
+- recommendations for new CNCF projects where there are clear use cases or where exploration is needed
```

Comment:
how about existing?

Reply:
How about we add this:
- review of upcoming project proposed for adoption and propose to CNCF according to interest

Reply:
How about "review the upcoming project applied for adoption and provide feedback to the CNCF TOC"

Reply:
Maybe "review the upcoming project applied for adoption and provide feedback to the CNCF TOC according to community interest" ?  I'd lie to know what we are evaluating on.

Reply:
"Provide environmental sustainment recommendations and optimizations to new and existing projects within the landscape and areas of gap in the environmental sustainment coverage of the landscape"

Reply:
will add to the PR to discuss.

Reply:
Updated PR with new wording from meeting today.